_data/whats-new.yml

```diff
@@ -4,8 +4,41 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon Apr 29 09:56:29 2019
+updated: Mon May  6 15:32:53 2019
```

Comment:
Looks like there is an extra space between the May and 6.

Reply:
yes, it is automatically generated by date converter

Reply:
fixed this in the generator https://github.com/dshevtsov/whats-new-on-devdocs/commit/4022a933dcc85b46e7ba5308fdc10cb88cc3b3e9